package.json

```diff
@@ -3,8 +3,15 @@
   "private": true,
   "devDependencies": {},
   "workspaces": {
-    "packages": ["packages/*"],
-    "nohoist": ["packages/*-subgraph"]
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@graphprotocol/graph-ts",
```

Comment:
We were using the nohoist wrong. We needed to select the specific node_module dependencies.